packages/renderer/src/lib/container/ContainerColumnImage.svelte

```diff
@@ -0,0 +1,11 @@
+<script lang="ts">
+import type { ContainerGroupInfoUI, ContainerInfoUI } from './ContainerInfoUI';
+
+export let object: ContainerInfoUI | ContainerGroupInfoUI;
+</script>
+
+{#if 'image' in object}
```

Comment:
```suggestion
{#if 'image' in object && typeof object.image === 'string'}
```

Reply:
Ditto to previous comment, there are only two types and image only exists in one of them, so what's the added benefit of checking the property type?

Reply:
type safety, never trust pattern

Reply:
As per other comment, I've added isContainerGroupInfoUI/isContainerInfoUI helper utils which do this.